main.py

```diff
@@ -62,40 +68,51 @@ def send_all_prices_message():
     curr_prices = get_all_prices_from_API()
     bot = telegram.Bot(token=BOT_TOKEN)
     try:
-        bot.sendMessage(chat_id=CHAT_ID, text="1€ = " + str(curr_prices[Currency.EURO]) + "mdl, 1$ = " + str(
-                                    curr_prices[Currency.US_DOLLAR]) + "mdl")
+        bot.sendMessage(chat_id=CHAT_ID, text="1€ = {}mdl, 1$ = {]mdl".format(
```

Comment:
you mistyped second {}

Reply:
brat, tu cand corectezi?

Reply:
Brat, scuz, trebu la univer de prashyt, a s ma uit azi seara.

Reply:
Solved